xc/device.ml

```diff
@@ -1854,7 +1860,8 @@ let stop ~xs ~qemu_domid domid  =
     stop_qemu ()
 
 let restore_vgpu (task: Xenops_task.t) ~xs restore_fd domid vgpu vcpus =
-	start_vgpu ~xs task ~restore_fd domid [vgpu] vcpus
+	debug "Called Dm.restore_vgpu";
+	start_vgpu ~xs task ~restore:true ~restore_fd domid [vgpu] vcpus
```

Comment:
`start_vgpu` doesn't have a `~restore` (yet). I also don't really like to add it, because it is completely redundant and potentially confusing.